modules/ROOT/pages/composefs.adoc

```diff
@@ -0,0 +1,50 @@
+= Composefs
+
+Fedora CoreOS introduced composefs enabled by default starting in Fedora 41. Composefs is an overlay filesystem where the data comes from the usual ostree deployement, and
+metadata are in the composefs file. The result is a truely read-only root (`/`) filesystem, increasing the system integrity and robustness,
+
+This is a first step towards a full verification of filesystem integrity, even at runtime.
+
+== What does it change ? 
+
+The main visible change will be that the root filesystem (/) is now small and full (a few MB, 100% used).
+
+== Known issues
+
+=== Kdump
+
+Right now, this prevents kdump from generating it's initramfs as it get confused by the read-only filesystem.
+If you want to use kdump and export kernels dumps to the local machine, composefs must be disabled.
+A workaround is to configure kdump with a remote target such as ssh or nfs.
+The kdump upstream developpers are working on a fix. We will update this page when the workaround is no longer needed.
+
+https://github.com/rhkdump/kdump-utils/pull/28
+
+=== Top-level directories
+
+Another consequence is that it is now impossible to create top-level direcories in `/`. Those are usually mount points.
```

Comment:
```suggestion
Another consequence is that it is now impossible to dynamically create top-level directories in `/`. A common use case for those top level directories is to use them as mount points. We recommend using sub directories in `/var` instead.
```

Reply:
`dynamically` suggest that there is a way around IMO.